c/busybox-1.22.0/realpath_true-no-overflow_true-valid-memsafety.c

```diff
@@ -283,8 +283,6 @@ signed int __main(signed int argc, char **argv)
     resolved_path=xmalloc_realpath(*argv);
     if(!(resolved_path == ((char *)NULL)))
     {
-      puts(resolved_path);
-      free((void *)resolved_path);
```

Comment:
I tried the busybox_sv_comp_impl.h way, but the patch gets really big, because it is also needed to include limits.h to every .c file (because of PATH_MAX). Including the definition only to the realpath and readlink means just to include few lines into 4 files (which can be done even manually), so IMHO that would be better, the more that we are going to get rid of the preprocessed files in the near future.